rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/HealthCheckController.java

```diff
@@ -1,24 +1,40 @@
 package gov.cms.qpp.conversion.api.controllers;
 
-import org.springframework.http.HttpStatus;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import gov.cms.qpp.conversion.api.model.HealthCheck;
+import gov.cms.qpp.conversion.api.services.VersionService;
+
 /**
  * Controller to simply respond with status 200 with a GET /health call.
  */
 @RestController
 @RequestMapping("/health")
 public class HealthCheckController {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)